packages/cli/src/schemas/schemagen.js

```diff
@@ -0,0 +1,20 @@
+import fs, { read } from "fs";
+import yaml from "js-yaml";
+import generateSchema from "generate-schema";
+
+try {
```

Comment:
This is a prebuild script

Reply:
I've never used a prebuild script before - we probably want to publish this to `npm` as a draft version and verify that before we publish it under the `latest` tag, to make sure it doesn't run for consumers of the tool

Reply:
Also, if this file is meant to run as a standalone tool, perhaps it would be better put under, e.g., `src/prebuild_scripts/schemagen.js`, to make it's purpose as a non-library script clear.

Reply:
It shouldn't be included; it's only run during the prebuild step to prepare things for actual building. That said, I'll make sure it's not included where it shouldn't be and move it to a better dir.